ouroboros-consensus/src/Ouroboros/Consensus/Util/CBOR.hs

```diff
@@ -146,46 +146,60 @@ readIncremental hasFS@HasFS{..} fp = withLiftST $ \liftST -> do
 -- and the error.
 --
 -- To be used by 'Ouroboros.Storage.ImmutableDB.Util.cborEpochFileParser''.
-readIncrementalOffsets :: forall m h a. (MonadST m, MonadThrow m)
```

Comment:
Let's keep both versions around for now. The original version (pre this PR) will be the one we'll eventually keep; the more specialized version is currently only needed for `cborEpochFileParser'`, but that will go and the ChainDB will take over that task. The more general version is then also applicable to the volatile DB.